.idea/codeStyleSettings.xml

```diff
@@ -5,6 +5,7 @@
       <value>
         <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="9999" />
         <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="9999" />
+        <option name="RIGHT_MARGIN" value="140" />
```

Comment:
https://github.com/docker-java/docker-java/blob/master/src/test/resources/checkstyle/checkstyle-config.xml#L91-L92
thanks. Added file only some time before, not fully filled.